### DISCUSSION TITLE
Plane stats

### DISCUSSION THREAD

**Discussion by Hadiiiiiiiiiiiiii** [FlareFlo/wt_data_sheets_wasm#181] [Ideas]
Have you guys tried to get plane stats from the FM files? I mean stuff like acceleration, climb/ turn rate etc based on fuel altitude and other factors.

**Comment by FlareFlo** [comment]
I believe this is possible, i will look into updating the prototype soon.

**Reply by Hadiiiiiiiiiiiiii** [reply]
i tried to do it myself and i got pretty far up until it came to prop thrust calculation/wave drag calculation, thurst for jets is very easy to get, it's linear between the points in the ThrustMax section of the fm file. I believe wave drag has something to do with the MachFactor, MachCrit, MachMax, MultMachMax, MultLineCoeff in each section of the aircrafts parts but I couldn't really find a way to link them all together.

**Reply by FlareFlo** [reply]
> i tried to do it myself and i got pretty far up until it came to prop thrust calculation/wave drag calculation, thurst for jets is very easy to get, it's linear between the points in the ThrustMax section of the fm file. I believe wave drag has something to do with the MachFactor, MachCrit, MachMax, MultMachMax, MultLineCoeff in each section of the aircrafts parts but I couldn't really find a way to link them all together.
> 

i unfortunately dont have much experience with said parameters, more specifically, the formulas needed to get the expected results :/
